service/common/src/main/java/org/apache/polaris/service/context/DefaultCallContextResolver.java

```diff
@@ -67,16 +62,6 @@ public CallContext resolveCallContext(
         .addKeyValue("queryParams", queryParams)
         .addKeyValue("headers", headers)
         .log("Resolving CallContext");
-    final Map<String, String> parsedProperties = parseBearerTokenAsKvPairs(headers);
-
-    if (!parsedProperties.containsKey(PRINCIPAL_PROPERTY_KEY)) {
-      LOGGER.warn(
-          "Failed to parse {} from headers ({}); using {}",
-          PRINCIPAL_PROPERTY_KEY,
-          headers,
-          PRINCIPAL_PROPERTY_DEFAULT_VALUE);
-      parsedProperties.put(PRINCIPAL_PROPERTY_KEY, PRINCIPAL_PROPERTY_DEFAULT_VALUE);
```

Comment:
We don't need set the default value for `PRINCIPAL_PROPERTY_KEY` anymore?

Reply:
This is dead code. The parsedProperties variable is discarded when the method returns.